src/geo/leaflet/leaflet-cartodb-layer-group-view.js

```diff
@@ -53,6 +53,13 @@ var LeafletCartoDBLayerGroupView = function (layerModel, leafletMap) {
   this.leafletLayer.on('loading', function (e) {
     self.trigger('loading');
   });
+
+  this.leafletLayer.on('tileerror', function (layer, coords, error) {
```

Comment:
Same described above, what about "encapsulating" this logic in other place, like the map code (map-view?). As you suggested, you could add more "map libraries" and you would not need to add specific logic for each one.